crates/criticaltrust/src/revocation_info.rs

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: The Ferrocene Developers
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::keys::KeyRole;
+use crate::signatures::Signable;
+use crate::NoRevocationsCheck;
+use serde::{Deserialize, Serialize};
+use time::macros::datetime;
+use time::OffsetDateTime;
+
+/// Holds hashes of revoked content which are included as a part of the [`KeysManifest`].
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RevocationInfo {
+    pub revoked_content_sha256: Vec<String>,
```

Comment:
Why are we storing the revoked content as a string? In every other manifest it's stored as a `Vec<u8>` with `#[serde(with = "crate::serde_base64")]` to encode it as base64.